src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -112,10 +113,7 @@ function getImageCrop(obj, imageStyle = null) {
     // 'https://{buildtype}.cms.va.gov/sites/default/files/', so add that here.
     // It doesn't matter what the build type is since it gets stripped out in convertDrupalFilesToLocal
     // so just use prod here
-    imageObj.image.derivative.url = `https://prod.cms.va.gov/sites/default/files/${url.replace(
-      '/img/',
-      '',
-    )}`;
+    imageObj.image.derivative.url = `${buckets.vagovprod}${url}`;
```

Comment:
Can we add a TODO comment here re: using the appropriate bucket for the buildtype (dev/staging/prod)?

Reply:
We will add this to a TODO